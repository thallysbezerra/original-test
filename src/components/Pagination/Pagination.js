import React from 'react';
import IconButton from '../IconButton/IconButton';

import './Pagination.scss';

const Pagination = () => (
	<div className="pagination">
		<IconButton
			color="secondary"
			disabled
			icon="chevron-left"
			onClick={() => alert('Produtos anteriores')}
			size="small"
		/>
		<div className="pagination__info">1 de 10</div>
		<IconButton
			color="secondary"
			icon="chevron-right"
			onClick={() => alert('Produtos posteriores')}
			size="small"
		/>
	</div>
);

export default Pagination;
