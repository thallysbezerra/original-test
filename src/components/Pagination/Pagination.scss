@import '../../css/Theme.scss';

.pagination {
	align-items: center;
	display: flex;
	justify-content: center;
	margin-top: 30px;

	&__info {
		color: $blue-dark;
		font-size: 14px;
		margin: 0 15px;
	}
}
