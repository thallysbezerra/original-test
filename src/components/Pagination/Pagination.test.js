import React from 'react';
import Pagination from './Pagination';

import renderer from 'react-test-renderer';

describe('Pagination UI', () => {
	it('<Pagination /> must have the snapshot version.', () => {
		const tree = renderer.create(<Pagination />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
