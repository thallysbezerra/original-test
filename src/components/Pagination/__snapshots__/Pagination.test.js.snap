// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination UI <Pagination /> must have the snapshot version. 1`] = `
<div
  className="pagination"
>
  <button
    className="icon-button icon-button--secondary icon-button--small "
    onClick={[Function]}
  >
    <i
      className="icon-button__icon fas fa-chevron-left"
    />
  </button>
  <div
    className="pagination__info"
  >
    1 de 10
  </div>
  <button
    className="icon-button icon-button--secondary icon-button--small "
    onClick={[Function]}
  >
    <i
      className="icon-button__icon fas fa-chevron-right"
    />
  </button>
</div>
`;
